### PR TITLE
add a CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by sup**
so i've made an extremely simple "guide" on how to get started on helping out. this is absolutely up for extension, and i'm happy to take any comments into consideration.

(also ping @zkat since i believe you wanted to see something like this as well)

### CODE REVIEW

**Comment by zkat** [comment]
My personal preference is to slow down on changes like these -- I definitely want to see this, but I want to wait until the meeting next week (#37) so we can first talk about how to use github effectively.

**Comment by sup** [comment]
@zkat sure, i'll just leave the PR open until then!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Even though we have got a charter, this WG is essentially still in its early
```
Comment (nebrius) [inline CONTRIBUTING.md]: Small grammar nit: can we drop the "got" so it's "Even though we have a charter"?

Comment (sup) [inline CONTRIBUTING.md]: @nebrius done!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Even though we have a charter, this WG is essentially still in its early
+stages.
+If you are interested in helping shape this it, have a look at the issues and PRs.
```
Comment (nebrius) [inline CONTRIBUTING.md]: Whoops, missed one more: can you change "in helping shape this it" to "in helping shape this WG"?

Comment (sup) [inline CONTRIBUTING.md]: oh man, how did i write this :smile: updating asap

**Comment by nebrius** [comment]
LGTM

**Pull request merged**